odoo/addons/base/models/ir_actions_report.py

```diff
@@ -499,9 +499,13 @@ def barcode(self, barcode_type, value, width=600, height=100, humanreadable=0, q
                 value = '0%s' % value
         try:
             width, height, humanreadable, quiet = int(width), int(height), bool(int(humanreadable)), bool(int(quiet))
+            bar_border = 4
+            if barcode_type == 'QR' and quiet:
+                bar_border = 0
```

Comment:
Alright, but just add a small comment so that we understand that in case we need to change it in a distant future. Something like " 'quiet' argument is ignored by reportlab for QR-barcodes. The generation of a "quiet-zone" around the barcode is instead managed by 'barBorder' "